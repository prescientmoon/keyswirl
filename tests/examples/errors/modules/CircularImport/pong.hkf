module exporting *

import errors.modules.CircularImport.ping
