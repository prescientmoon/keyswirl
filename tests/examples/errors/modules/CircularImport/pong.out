[error CircularImport]: Module "errors.modules.CircularImport.pong" implicitly imports itself
     +--> errors.modules.CircularImport.ping@3:8-3:42
     |
   3 | import errors.modules.CircularImport.pong
     :        ^---------------------------------
     :        `- This is where the cycle closes is
-----+

[error ModuleNotFound]: Module "errors.modules.CircularImport.pong" not found
     +--> errors.modules.CircularImport.ping@3:8-3:42
     |
   3 | import errors.modules.CircularImport.pong
     :        ^---------------------------------
     :        `- This module does not exist
-----+
